Merged develop and fixed gradle build

rohitvvv opened the pull request:
Synced up master and develop. Fixed the gradle build.

rohitvvv committed:
Adding Gradle Support for GitFx

rohitvvv committed:
Merge pull request #64 from jughyd/issue63

Adding Gradle Support for GitFx

sagarpatel26 committed:
added AOP

rohitvvv committed:
Merge pull request #67 from sagarpatel26/develop

Added AOP Logging

SaiUpadhyayula committed:
Issue #69

Fixed NullPointer Exception when clicked on Cancel button in the FileDialog

SaiUpadhyayula committed:
Added Unit Test to test the FileDialog functionality

SaiUpadhyayula committed:
Added Unit Test to test the FileDialog functionality

SaiUpadhyayula committed:
Added Unit Test to test the FileDialog functionality

rohitvvv committed:
Merge pull request #70 from SaiUpadhyayula/develop

Issue #69

Commit:
Merging master and develop

Merge branch 'develop' of https://github.com/jughyd/GitFx

Commit:
Adding aspectj dependency to build.gradle

rohitvvv commented:
@jbuddha Please review.

jbuddha commented on .gitignore:
```diff
@@ -30,7 +31,9 @@ nbactions.xml
 
 #folder conatining logs
 logs/
+>>>>>>> 834b8d3f0dd091cf4608cb6b496ff469a1905652
```
is this necessary? seems like a byproduct of resolving conflicts.

jbuddha commented on src/main/java/io/github/gitfx/dialog/GitFxDialog.java:
```diff
@@ -86,17 +87,13 @@ public String gitFxInformationListDialog(String title, String header, String lab
-                }
-         });
+                (observable, oldValue, newValue) -> {
+//[LOG]                  logger.debug(newValue.toString());
```
this listener may be removed as this is not doing anything

Commit:
Cleanup GitFxDialog and .gitignore
Removed unused listner for an Information Dialog
Removed merge conflict marker from .gitignore
